src/components/Table/useClassNames.ts

```diff
@@ -28,8 +29,18 @@ export function useClassNames() {
       cell: {
         wrapper: generateForCell(),
       },
+      tableReel: {
+        wrapper: generateForTableReel(),
```

Comment:
wrapper が Shadow で、inner が Wrapper なのかも?
（Wrapper の名前を見直してもよさそう）

```suggestion
        wrapper: generateForTableReel(),
        inner: generateForTableReel('inner'),
```

Reply:
この辺のルールって今どうしてるんだ（一旦これでいく）

Reply:
あまり、この仕組み自体に納得しないまま導入が始まったのもあって、何も決まってないです。
みんな雰囲気で書いている。